app/views/welcome/index.html.erb

```diff
@@ -19,8 +19,14 @@ to add something
 </div>
 
 <h2>
-  Hello everyone!!! This is my part.
+  Hello everyone!!! This is bourak.
```

Comment:
This replaces the existing greeting with your own. What I would like to see instead, is an __additional__ greeting.
Please add a new commit.